pytorch_lightning/trainer/trainer.py

```diff
@@ -189,7 +189,10 @@ def __init__(
                 GPUs are configured to be in "exclusive mode", such
                 that only one process at a time can access them.
 
-            num_tpu_cores: How many TPU cores to train on (1 or 8).
+            tpu_cores: How many TPU cores to train on (1 or 8) / Single TPU to train on [1]
```

Comment:
just 1 or 8, nothing in between?

Reply:
Yes. The `nprocs` argument for `xm.spawn` supports either 1 or max number of devices.
Source: https://pytorch.org/xla/release/1.5/index.html#torch_xla.distributed.xla_multiprocessing.spawn